test/draw/RasterCanvasTest.ooc

```diff
@@ -55,7 +55,7 @@ RasterCanvasTest: class extends Fixture {
 			output := "test/draw/output/RasterCanvas_Yuv420.png"
 			image := RasterYuv420Semiplanar open(this inputFlower)
 			for (i in 0 .. 30) {
-				image canvas pen = Pen new(ColorBgr new((i % 10) * 25, (i % 5) * 50, (i % 3) * 80))
+				image canvas pen = Pen new(ColorRgb new((i % 3) * 80, (i % 5) * 50, (i % 10) * 25))
```

Comment:
These looks like actual RGB colors since they would represent invalid YUV colors and they are drawn on the whole YUV image.